src/Lib/Elastic/Filter/Params/AttributesParams.php

```diff
@@ -33,10 +34,16 @@ class AttributesParams extends BasicParams implements ElasticParamElementInterfa
      *
      * @throws \Exception
      */
-    public function __construct($code, array $values, string $operator = null)
+    public function __construct(
+        $code, 
+        array $values, 
+        string $operator = null, 
+        string $field = '_attr.valueId'
```

Comment:
_attr it is the constant part for an attribute in mapping, and it is better just to operate with real attribute fields in the mapping without prefix.

Reply:
`_attr` prefix isn't something constant and also could be changed in future.